Frontend/feature/post edit

berfinsimsekk opened the pull request:
Post owners can edit their post's title and description with an edit button which is on the post detail page. Only post owners can edit their posts, other users don't see the edit button. 
After editing the post, you should refresh the page to see your changes. Please review and let me know if there is a problem.

LehabuL reviewed (changes requested):
I think we should move the edit button out of the box and position it either on the right or left side. Also it would be great to adjust the size of the edit form

berfinsimsekk commented:
I removed unnecessary fields from edit form so its size is smaller. I think edit button's position is better in the box but I changed its orientation. Let me know whether is okay for you too. Thanks

LehabuL reviewed (approved):
I have tested the functionality and it looks good. Also it is great to see that the annotationsdisappear oncec the description part is updated, thanks for your efforts.
